index.html

```diff
@@ -1301,6 +1301,33 @@ <h2><code>JsonSchemaCredential</code></h2>
                </p>
             </section>
          </section>
-      </section> 
+      </section>
+      <section class="informative">
+         <h2>Revision History</h2>
+         <p>
+            This section contains the substantive changes that have been made to
+            this specification over time.
+         </p>
+         <p>
+            Changes since the
+            <a href="https://www.w3.org/TR/2023/WD-vc-json-schema-20230915/">
+            First Public Working Draft</a>:
+         </p>
+         <ul>
+            <li>Renamed `JsonSchema2023` to `JsonSchema`</li>
+            <li>Renamed `CredentialSchema2023` to `JsonSchemaCredential`</li>
+            <li>Added the `jsonSchema` type for usage within a `credentialSubject`</li>
+            <li>Provided guidance on which media types should be used</li>
+            <li>Added section on addressing PII in schemas</li>
+            <li>Added section on using Content Integrity Protection</li>
+            <li>Added text on accessibility and internationalization</li>
+            <li>Added guidance on processing JSON Schemas</li>
+            <li>Required the use of [[JSON-SCHEMA-2020-12]]</li>
+            <li>Aligned term and vocabulary definitions with the [[VC-DATA-MODEL-2.0]]</li>
+            <li>Added support for YAML representations of JSON Schemas</li>
+            <li>Addressed privacy review concerns, such as advocating for the usage of OHTTP, and encouraging data minimization</li>
+            <li>Remove superfluous normative references</li>
+         </ul>
```

Comment:
```suggestion
         <section>
           <h3>
              Changes since the
              <a href="https://www.w3.org/TR/2023/WD-vc-json-schema-20230915/">
              First Public Working Draft</a>:
           </h3>
           <ul>
              <li>Renamed `JsonSchema2023` to `JsonSchema`</li>
              <li>Renamed `CredentialSchema2023` to `JsonSchemaCredential`</li>
              <li>Added the `jsonSchema` type for usage within a `credentialSubject`</li>
              <li>Provided guidance on which media types should be used</li>
              <li>Added section on addressing PII in schemas</li>
              <li>Added section on using Content Integrity Protection</li>
              <li>Added text on accessibility and internationalization</li>
              <li>Added guidance on processing JSON Schemas</li>
              <li>Required the use of [[JSON-SCHEMA-2020-12]]</li>
              <li>Aligned term and vocabulary definitions with the [[VC-DATA-MODEL-2.0]]</li>
              <li>Added support for YAML representations of JSON Schemas</li>
              <li>Addressed privacy review concerns, such as advocating for the usage of OHTTP, and encouraging data minimization</li>
              <li>Remove superfluous normative references</li>
           </ul>
         </section>
```